apps/web/components/message-wrapper.tsx

```diff
@@ -0,0 +1,48 @@
+'use client'
+
+import { usePathname, useSearchParams } from 'next/navigation'
+import React, { useEffect, useState } from 'react'
+
+type MessageWrapperProps = {
+  children: React.ReactNode
+  snowflakeId: string
+}
+export const MessageWrapper = ({
+  children,
+  snowflakeId,
+}: MessageWrapperProps) => {
+  const searchParams = useSearchParams()
+  const [isHighlighted, setIsHighlighted] = useState(false)
+  const pathname = usePathname()
+
+  useEffect(() => {
+    if (!searchParams || searchParams.has(snowflakeId)) return
+    const searchID = searchParams.get('replyID')
+    let timeout: NodeJS.Timeout
+    if (searchID && searchID === snowflakeId) {
+      setIsHighlighted(true)
+      timeout = setTimeout(() => {
+        setIsHighlighted(false)
+        window.history.replaceState(null, '', pathname)
+      }, 1000)
+    }
+    return () => {
+      if (timeout) clearTimeout(timeout)
+    }
+  }, [searchParams, snowflakeId])
```

Comment:
This seems to be missing the pathname
```suggestion
  }, [searchParams, snowflakeId, pathname])
```

Reply:
Updated to not use pathname